Documentation/Usage.md

```diff
@@ -115,204 +115,254 @@ url.
+The `link "cairo"` line specifies the linker flag used to link the system library, roughly equivalent to `-lcairo` in C.
 
-Create a directory called `Clibgit` next to the `example` directory and
-initialize it as a package that builds a system module:
+In the `cairo.h` file, add the following line:
 
-    example$ cd ..
-    $ mkdir Clibgit
-    $ cd Clibgit
-    Clibgit$ swift package init --type system-module
+```c
+#include <cairo.h>
+```
 
-This creates `Package.swift` and `module.modulemap` files in the directory.
-Edit `Package.swift` and add `pkgConfig` parameter:
+This file functions just like a normal C header, where the `<>` brackets tell `clang` to look for the `cairo.h` header installed by your system in the usual locations. While it’s possible to reference the Cairo header directly in the `module.modulemap` file, including it through a local shim prevents you from having to specify an exact path to it. Many popular C libraries also allow (and even require) you to perform some customization at the inclusion site, so the local header file gives you a place to do this.
 
-```swift
-import PackageDescription
+Next, we have to tell the package manager about the system library module. Go into your `Package.swift` and add a `systemLibrary` target. Don’t forget to specify it as a dependency of your Swift `example` module:
 
+```swift
 let package = Package(
-    name: "Clibgit",
-    pkgConfig: "libgit2"
+    name: "example",
+    targets: [
+        .systemLibrary(name: "cairo", pkgConfig: "cairo"),
+        .target(name: "example", dependencies: ["cairo"])
+    ]
 )
 ```
 
-The `pkgConfig` parameter helps SwiftPM in figuring out the include and library
-search paths for the system library.  Note: If you don't want to use the `pkgConfig`
-parameter you can pass the path of a directory containing the library using the
-`-L` flag in commandline when building your app:
+The `pkgConfig:` parameter specifies the name of the system package that the package manager will ask the system `pkg-config` tool about. Without this information, `clang` may not know where to look for the installed Cairo headers. You can see what `pkg-config` will tell the package manager by running the tool directly in the terminal:
 
-    example$ swift build -Xlinker -L/usr/local/lib/
+```bash
+$ pkg-config --cflags cairo
+```
+```
+-I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include 
+-I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 
+-I/usr/include/freetype2 -I/usr/include/libpng16
+```
 
-Edit `module.modulemap` so it consists of the following:
+Unlike Swift targets, the `name:` parameter in a system library target is only used by the package manager; the name of the module as seen by Swift is entirely determined by the `module cairo` declaration in the `module.modulemap` file. You are free to use a different name for the module within the package description:
```

Comment:
I think we should remove the below example and explicitly call out that doing this would be against best practices.